api/topn_agg.md

```diff
@@ -0,0 +1,71 @@
+# topn_agg() <tag type="toolkit" content="Toolkit" /><tag type="experimental" content="Experimental" />
+Produces an aggregate that can be passed to the [`topn` function][topn] to
+calculate the `n` most-frequent values in a column.
+```sql
+topn_agg (
+    FIXME INTEGER,
```

Comment:
Didn't know the name of this parameter and just guessed at the type

Reply:
It is an `INTEGER` and it is the `n` referred to in the paragraph above, so I would suggest calling it `n` here as well.